### DISCUSSION TITLE
Doris 2.3 执行SQL后的耗时和Query Profile中的耗时不一致。

### DISCUSSION THREAD

**Discussion by DA1OOO** [apache/doris#29024] [A - General / Q&A]
执行完SQL后，WEB界面中显示的耗时同MySQL Client中显示的执行耗时（e.g 大概9s），然后查query profile显示耗时只有154ms，请问这是正常现象还是bug。如果是正常现象，那么这两个耗时分别具体是哪个阶段的耗时呢？

**Comment by DA1OOO** [comment]
关闭pipeline之后，时间正常。

**Comment by LemonLiTree** [comment]
正常，实际执行以profile里显示为准，这个时间差是在收集profile信息

**Reply by DA1OOO** [reply]
关了pipeline之后 时间没差这么多 https://github.com/apache/doris/issues/29041

**Reply by DA1OOO** [reply]
profile收集时间这么长的话，对正常查询耗时影响很大。

**Reply by LemonLiTree** [reply]
正常情况下不需要开profile
